packages/tocco-ui/src/FormattedValue/typeFormatters/NumberFormatter.js

```diff
@@ -4,18 +4,20 @@ import {FormattedNumber} from 'react-intl'
 
 import Typography from '../../Typography'
 
-const NumberFormatter = props => (
-  <Typography.Span>
-    <FormattedNumber
-      value={props.value}
-      style="decimal"
-      maximumFractionDigits={0}
-    />
-  </Typography.Span>
-)
+const NumberFormatter = props => {
+  return (
+    <Typography.Span>
+      <FormattedNumber
+        value={props.value}
+        style="decimal"
+        minimumFractionDigits={2}
+      />
+    </Typography.Span>
+  )
+}
 
 NumberFormatter.propTypes = {
-  value: PropTypes.number
+  value: PropTypes.number.isRequired
```

Comment:
i dont think we sould set a value as required. a value should and can be empty/null